cmd/earthly/main.go

```diff
@@ -805,6 +806,19 @@ func newEarthlyApp(ctx context.Context, console conslogging.ConsoleLogger) *eart
 				},
 			},
 		},
+		{
+			Name:   "config",
+			Usage:  "Edits your Earthly configuration file",
+			Hidden: true, // Experimental.
+			Action: app.actionConfig,
+			Flags: []cli.Flag{
+				&cli.BoolFlag{
+					Name:        "dry-run",
+					Usage:       "Print the changed config file to the consile instead of writing it out",
+					Destination: &app.configDryRun,
+				},
+			},
+		},
```

Comment:
Maybe we need a command to just print your config? But that command is also named `cat ~/.earthly/config.yml`...

Reply:
I think it would be nice nevertheless - similar to how you can inspect git config. But probably not as high in priority as setting values.